securedrop_client/api_jobs/downloads.py

```diff
@@ -264,4 +268,7 @@ def call_decrypt(self, filepath: str, session: Session = None) -> None:
         '''
         fn_no_ext, _ = os.path.splitext(os.path.splitext(os.path.basename(filepath))[0])
         plaintext_filepath = os.path.join(self.data_dir, fn_no_ext)
-        self.gpg.decrypt_submission_or_reply(filepath, plaintext_filepath, is_doc=True)
+        plaintext_path, original_filename = self.gpg.decrypt_submission_or_reply(
```

Comment:
Yeah, or maybe just stop returning it. There was a comment suggesting that before I got here, since the returned `plaintext_path` is only ever used in tests. I think I was going for minimally invasive by just adding the extra return value, but if it's not needed...